src/tools/cargotest/main.rs

```diff
@@ -48,6 +58,23 @@ const TEST_REPOS: &[Test] = &[
         // Only test Stylo a.k.a. Quantum CSS, the parts of Servo going into Firefox.
         // This takes much less time to build than all of Servo and supports stable Rust.
         packages: &["selectors"],
+        features: None,
+        manifest_path: None,
+    },
+    Test {
+        name: "diesel",
+        repo: "https://github.com/diesel-rs/diesel",
+        sha: "91493fe47175076f330ce5fc518f0196c0476f56",
+        lock: None,
+        packages: &[],
+        // Test the embeded sqlite variant of diesel
+        // This does not require any dependency to be present,
+        // sqlite will be compiled as part of the build process
+        features: Some(&["sqlite", "libsqlite3-sys/bundled"]),
```

Comment:
How long does this take to compile?

Reply:
A clean build takes ~1min on my not really powerful laptop and maybe ~1 minutes to run all tests.

Reply:
Yeah, that sounds fine, thanks :)